### PR TITLE
Toaster Update Recent-one

### PR DESCRIPTION
**Pull request by shreya-fyle1**

### CODE REVIEW

File: src/app/core/services/payment-modes.service.ts [hunk 1]
```diff
@@ -76,4 +82,13 @@ export class PaymentModesService {
       })
     );
   }
+
+  showInvalidPaymentMode() {
```
Comment (Dimple16) [inline src/app/core/services/payment-modes.service.ts:86]: showInvalidPaymentMode -> `showInvalidPaymentModeToast`

**Review by Dimple16 — CHANGES_REQUESTED** [review]
add screenshots of before and after the change

**Comment by shreya-fyle1** [comment]
Screenshot (508)
Before fixing

**Comment by shreya-fyle1** [comment]
Screenshot (507)
After fixing

**Comment by shreya-fyle1** [comment]
Screenshot (506)
After fixing

**Review by Dimple16 — APPROVED** [review]

**Pull request merged**